muranotest/Dockerfile

```diff
@@ -13,9 +16,10 @@ RUN python setup.py install
 RUN pip install nose tempest-lib
 WORKDIR /opt/murano
 RUN pip install -r test-requirements.txt
+COPY requirements.txt /opt/murano/requirements2.txt
```

Comment:
It is used to avoid the following error:
ValueError: ("Expected ',' or end-of-list in", "Routes!=2.0,!=2.1,>=1.12.3;python_version=='2.7'", 'at', ";python_version=='2.7'")